proto/buf.gen.ui.yaml

```diff
@@ -4,6 +4,6 @@ managed:
   go_package_prefix:
     default: github.com/rilldata/rill
 plugins:
-  - plugin: buf.build/bufbuild/es
+  - plugin: buf.build/bufbuild/es:v1.10.0
```

Comment:
This change should not be passed as you've already passed generated proto sources.

Reply:
Anyone else generating will face the same issue if we dont pin the version.